### DISCUSSION TITLE
May I ask where the 'languages. toml' file is located under Windows.

### DISCUSSION THREAD

**Discussion by sgj123456** [helix-editor/helix#7418] [Q&A] [answered]
My system is Windows 11, and I cannot find the location of my 'languages. toml' file

**Comment by the-mikedavis** [comment, marked as answer]
See https://docs.helix-editor.com/master/languages.html

The default languages.toml is compiled into the `hx` binary. You shouldn't have any other `languages.toml` files unless you create them. You can create a global one on Windows at `%AppData%\helix\languages.toml` or one local to a project in the root of that project in a `.helix\languages.toml` file.
